multiqc/utils/search_patterns.yaml

```diff
@@ -398,6 +398,9 @@ ngsderive/instrument:
 ngsderive/readlen:
   contents_re: "(.*)File(.*)Evidence(.*)MajorityPctDetected(.*)ConsensusReadLength"
   num_lines: 1
+optitype:
+  contents_re: "(.*)A1(.*)A2(.*)B1(.*)B2(.*)C1(.*)C2(.*)Reads(.*)Objective"
```

Comment:
This style of search pattern is apparently evil in terms of run time (see https://github.com/ewels/MultiQC/pull/1370#discussion_r588935271). Please just use `contents` and a pasted string if possible.

Reply:
Also added to use `contents` now 👍🏻